packages/core/mesh/network-manager/package.json

```diff
@@ -57,13 +57,12 @@
     "@dxos/protocols": "workspace:*",
```

Comment:
Is the "browser" field above up-to-date?

I also think we should switch to [Subpath Imports](https://nodejs.org/api/packages.html#subpath-imports) for all partform-specific functionality it's supported natively everywhere without requiring extra tooling.